index.js

```diff
@@ -28,5 +28,5 @@ function appendCSS(name){
   document.getElementsByTagName("head")[0].appendChild(stylesheet)
 }
 
-appendCSS('//gh-widget.oyanglul.us/main.css');
+appendCSS('/main.css');
```

Comment:
I changed to point to the local file for testing. You might want to change it back to the original url if you accept the pull request.